cognite/client/_api/datapoint_tasks.py

```diff
@@ -739,11 +744,18 @@ def store_partial_result(self, res: DataPointListItem) -> Optional[List[Splittin
             return self._split_self_into_new_subtasks_if_needed(last_ts)
         return None
 
-    def _create_subtasks_idxs(self, n_new_tasks: int) -> Iterable[Tuple[float, ...]]:
+    def _create_subtasks_idxs(self, n_new_tasks: int) -> Generator[Tuple[float, ...], None, None]:
```

Comment:
I think the type hint for generators is not very good, thus I think the old one should stay, if it is possible configure the linter to do so. 

Note I am not alone https://stackoverflow.com/questions/42531143/how-to-type-hint-a-generator-in-python-3